h/services/group.py

```diff
@@ -34,6 +34,8 @@ def create_private_group(self, name, userid, description=None):
         :param name: the human-readable name of the group
         :param userid: the userid of the group creator
         :param description: the description of the group
+        :param organization: the organization that this group belongs to in the form
```

Comment:
`:type organization: h.models.Organization` would be a clearer way of indicating the type rather than "in the form of a model object. See existing uses of `:type` in the code.